site/templates/tv/detail.jet

```diff
@@ -31,8 +31,10 @@
         {{end}}
         <div class="meta-detail-content">
           <h1>{{tvseason.GetTitle(i18n)}}</h1>
-          <div class="meta-detail-tagline">
-            {{yield metaItemTagline() tvseason}}
+          <div class="meta-detail-tagline-and-classification">
```

Comment:
What does this do other than make the tagline elements 3 containers deep instead of 2?

Reply:
- Adds consistency with the film page which has this div
- There's a margin that's applied via this class that appears on the film detail page (see [this bug card](https://dev.azure.com/S72/SHIFT72/_boards/board/t/Web%20team/Stories/?workitem=9218))
- Then I've adjusted the margin for both film and tv.

I often come across very similar and repeat code in the template (and css), starting to like your start from scratch idea

Reply:
If this adds consistency then 👍 .